nixos/modules/config/system-path.nix

```diff
@@ -107,12 +107,8 @@ in
         "/etc/gtk-3.0"
         "/lib" # FIXME: remove and update debug-info.nix
         "/sbin"
-        "/share/applications"
-        "/share/desktop-directories"
+        "/share"
```

Comment:
Why this? The rest of the `share` subdirectories in `pathsToLink` would be pointless then. I think we want to avoid `"/share"` (principle of least privilege or something).

Reply:
Hm, I think I was confused by the need for `/etc/xdg` but not `/share` - I think you're right that it will do the right thing because we typically include subdirectories of `/share` explicitly.